client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,215 @@
+                </b-form-group>
+            </b-col>
+            <b-col v-else-if="display == 'checkboxes'">
+                <!-- Select multiple from checkboxes -->
+                <b-form-group>
+                    <b-form-checkbox-group
+                        v-model="currentValue"
+                        :options="formattedOptions"
+                        value-field="value"
+                        text-field="label"
+                        stacked>
+                    </b-form-checkbox-group>
+                </b-form-group>
+            </b-col>
+            <b-col v-else>
+                <!-- Multiple select from drop down -->
+                <multiselect
+                    v-if="multiple"
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    :multiple="true"
+                    placeholder="Select value"
+                    deselect-label=""
+                    select-label=""
+                    track-by="value"
+                    label="label">
+                </multiselect>
+                <!-- Single select from drop down -->
+                <multiselect
+                    v-else
+                    v-model="currentValue"
+                    :options="formattedOptions"
+                    deselect-label=""
+                    select-label=""
+                    label="label"
+                    track-by="value">
+                    {{ currentValue }}
+                </multiselect>
+            </b-col>
+        </b-row>
+    </div>
+</template>
+
+<script>
+import Multiselect from "vue-multiselect";
+
+export default {
+    components: {
+        Multiselect,
+    },
+    props: {
+        value: {
+            default: null,
+        },
+        defaultValue: {
+            type: [String, Array],
+            required: false,
+            default: null,
+        },
+        options: {
+            type: Array,
+            required: true,
+            default: undefined,
+        },
+        multiple: {
+            type: Boolean,
+            required: true,
+            default: false,
+        },
+        display: {
+            type: String,
+            required: false,
+            default: null,
+        },
+        optional: {
+            type: Boolean,
+            required: false,
+            default: null,
+        },
+    },
+    data() {
+        return {
+            // Currently never set?
+            errorMessage: "",
+        };
+    },
+    computed: {
+        formattedOptions() {
+            const formattedOptions = [];
+            for (let i = 0, len = this.options.length; i < len; i++) {
+                formattedOptions[i] = {
+                    label: this.options[i][0],
+                    value: this.options[i][1],
+                    default: this.options[i][2],
+                };
+            }
+            if (this.multiple == false && this.optional == true) {
```

Comment:
Should be written as `!this.multiple && this.optional`.

Reply:
Agreed - change made.